topics/littlelink-custom/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: littlelinkcustom
```

Comment:
```suggestion
```

The alias topic isn't actually used on any repositories yet! We tend to only add aliases (or curate topics) when they're being used.